bitbake/lib/bb/ui/buildinfohelper.py

```diff
@@ -1711,6 +1711,8 @@ def scan_image_artifacts(self):
         if image_fstypes != None:
             image_types_str = image_fstypes.strip()
             image_file_extensions = re.sub(r' {2,}', ' ', image_types_str)
+            # MEL specific addition to for wic.bmap file
+            image_file_extensions += " wic.bmap"
```

Comment:
As this is a poky change, this should be suitable for upstream, not hardcoded for mel, otherwise we're stuck with the fork indefinitely. Can toaster not use IMAGE_FSTYPES from the image recipe rather than the config metadata? It's already listed there.